src/utils/getPage.js

```diff
@@ -8,6 +8,7 @@ import _ from 'lodash';
  * @return {Object}
  */
 export default function getPage(pages, urlPath) {
+    
```

Comment:
Please undo this change

Reply:
Okay, I edit and updated the lockfile version  1 to current version 2